src/mocks/offer.ts

```diff
@@ -0,0 +1,25 @@
+import Randomizer from '../utils/randomizer';
+import { POINTS_TYPES } from '../const';
+import { OFFERS } from './const.mock';
+import type { InnerOffer, Offer } from '../types/offer';
+
+const RANDOM_PRICE_FROM = 10;
+const RANDOM_PRICE_TO = 10;
+
+const RANDOM_OFFER_FROM = 0;
+const RANDOM_OFFER_TO = 5;
```

Comment:
[Б16.](https://up.htmlacademy.ru/ecmascript/23/criteries#b16) Сложные составные константы собираются в перечисления Enum.
```suggestion
const RandomPrice = {
  FROM: 10,
  TO: 10,
};

const RandomOffer = {
  FROM: 0,
  TO: 5,
};
```

хотя в тс можно так
```suggestion
const enum RandomPrice {
  FROM = 10,
  TO = 10,
}

const enum RandomOffer {
  FROM = 0,
  TO = 5,
}
```

Reply:
Спасибо ) Запарился и упустил вчера этот момент.